_pages/en_US/installing-unlaunch.md

```diff
@@ -71,7 +71,7 @@ We will also reconfigure nds-bootstrap to launch into TWiLight Menu++ (instead o
 1. Navigate to `OPTIONS`, and look at the available options
    - <kbd>A</kbd> + <kbd>B</kbd> is hardcoded to launch into Unlaunch's menu, and as such cannot be changed
    - The `NO BUTTON` and `BUTTON A/B/X/Y` options can be set however you like and will choose what your DSi loads at boot depending on which buttons are held. You can select any DSiWare, Homebrew, the Slot-1 card, or Unlaunch's file menu
-      - For TWiLight Menu++, select the `TWiLight Menu++` option where `boot.nds` is shown on the bottom screen
+      - For TWiLight Menu++, select the `TWiLight Menu++` option where `BOOT.NDS` is shown on the bottom screen
       - For the original DSi Menu, select `Launcher`
    - `LOAD ERROR` is what your DSi will load if loading what you have set fails, such as the SD card not being inserted. If you would like to load into the DSi Menu, then set this to `Launcher`
```

Comment:
> If you would like to load into the DSi Menu, then set this to `Launcher`

The way this sentence is written makes it seem that you must set that and only that in order to boot into the DSi Menu. With context, this sentence could probably be removed here considering what's said on the line above.

Reply:
Hmm, I don't really see how that makes it sound like that's the only way to load the DSi Menu tbh... But I do agree its not really needed considering the line above so you can remove that if you want